index.html

```diff
@@ -244,7 +370,8 @@ <h2>About me</h2>
                                 </div>
```

Comment:
- [ ] Kindly increase the font weight of this text to ensure a pixel-perfect design. 😉

![image](https://user-images.githubusercontent.com/41163571/230712616-ad4ec99a-27b6-4a0f-a4d3-a02e35de2bf5.png)

Reply:
Done.